### PR TITLE
Remote Audit LOG

### PR DESCRIPTION
**Pull request by arulkumar9690**
SUMMARY
Logging - Module Enhancement

GitHub Issues
N/A.

ISSUE TYPE
Feature Pull Request
Added "audit" message-type option

COMPONENT NAME
sonic_logging

ADDITIONAL INFORMATION

OUTPUT

Regression test HTML report:
[Remote_audit_LOG.pdf](https://github.com/user-attachments/files/16537600/Remote_audit_LOG.pdf)

Checklist:
[x] I have performed a self-review of my own code to ensure there are no formatting, linting, or security issues
[x] I have verified that new and existing unit tests pass locally with my changes
[x] I have not allowed coverage numbers to degenerate
[x] I have maintained at least 90% code coverage
[x] I have commented my code, particularly in hard-to-understand areas
[x] I have made corresponding changes to the documentation
[x] I have added tests that prove my fix is effective or that my feature works
[x] I have maintained backward compatibility or have provided any relevant "breaking_changes" descriptions in a "fragment" file in the "changelogs/fragments" directory of this repository.
[x] I have provided a summary for this PR in valid "fragment" file format in the "changelogs/fragments" directory of this repository branch. Reference : [Ansible Change Log Document](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)

How Has This Been Tested?
Ran lots of unit testing and regression testing on some SONIC target systems.

### CODE REVIEW

File: plugins/modules/sonic_logging.py
Comment (kerry-meyer) [inline plugins/modules/sonic_logging.py:1]: There is a merge conflict for this file. (The "deleted" example has a merge conflict with the changes made for adding TLS logging with https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/423.)

Please resolve this conflict.

**Review by kerry-meyer — CHANGES_REQUESTED** [review]
The proposed changes and test results look good.

Before merging this PR, there is a conflict that needs to be resolved:

There is a merge conflict for the "module" file. (The "deleted" example has a merge conflict with the changes made for adding TLS logging with https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/423.)

Please resolve this conflict to proceed with approval and merging.

**********
While you are doing the conflict resolution, please also correct the text for the "overridden" example to use the correct state ("override" instead of "replace" at line 184).

File: plugins/modules/sonic_logging.py
Comment (arulkumar9690) [reply plugins/modules/sonic_logging.py:1]: Resolved the merge conflicts.

**Review by kerry-meyer — APPROVED** [review]
All proposed changes and corresponding test results look good.

Approved.

**Pull request merged**